Figgle.Generator/Figgle.Generator.csproj

```diff
@@ -27,17 +33,20 @@
   </ItemGroup>
 
   <ItemGroup>
-    <None Include="$(OutputPath)\Figgle.dll"           Pack="true" PackagePath="analyzers\dotnet\cs" />
-    <None Include="$(OutputPath)\Figgle.pdb"           Pack="true" PackagePath="analyzers\dotnet\cs" />
+    <None Include="$(OutputPath)\Figgle.dll" Pack="true" PackagePath="analyzers\dotnet\cs" />
+    <None Include="$(OutputPath)\Figgle.pdb" Pack="true" PackagePath="analyzers\dotnet\cs" />
+    <None Include="$(OutputPath)\Figgle.Fonts.dll" Pack="true" PackagePath="analyzers\dotnet\cs" />
+    <None Include="$(OutputPath)\Figgle.Fonts.pdb" Pack="true" PackagePath="analyzers\dotnet\cs" />
```

Comment:
Do we have to pack `Figgle.Fonts.dll` into the source generator assembly, or can it be obtained via `PackageReference`?